e2e-tests/e2e-legacy/sparql-editor/actions/inferred-sameas.spec.js

```diff
@@ -2,16 +2,13 @@ import {SparqlEditorSteps} from "../../../steps/sparql-editor-steps";
 import {YasqeSteps} from "../../../steps/yasgui/yasqe-steps";
 import {RepositoriesStub} from "../../../stubs/repositories-stub";
 
-/**
- * TODO: Fix me. Broken due to migration (Error: beforeEach)
- */
-describe.skip('Expand results over owl:sameAs', () => {
+describe('Expand results over owl:sameAs', () => {
```

Comment:
Are those ok now because of this MR?

Reply:
Yes, without changes in this MR, the YASGUI is not loaded into the test, which is why the tests failed.